src/js/lib/client/entitlements.mac.plist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This file is required for the macOS notarisation. Since we require a hardened runtime, we need
```

Comment:
" Since we require a hardened runtime" - this should be explained. Cool that it has the link to the description of what hardened runtime is, but it would still be cool to just describe in 1 sentence what hardened runtime is and in another why we require it. Links should be additional explanation, not part of the core explanation - so basically, after reading a comment we should more or less understand why the design choice was made.